docs/source/examples/input_schemas.rst

```diff
@@ -0,0 +1,119 @@
+Action Provider Input Schemas
+=============================
+
+We support user-written JSONSchema and Pydantic_ based schema defintions. In
+this document we demonstrate how either defintion can be created and loaded into
+an ActionProvider to provide input documentation and validation. In many cases,
+it's easier to work with Pydantic models to define JSON input.
+
+JSONSchema
+^^^^^^^^^^
+
+A typical schema definition may look like:
+
+.. code-block:: JSON
+
+    {
+        "$id": "https://automate.globus.org/skeleton_action_provider.input.schema.json",
```

Comment:
Ah so that's what the field gives us. I think there's a way to do that on the pydantic BaseModels. I'll need to think what the best way to add that to user input models will be. Do you think this applies more to one JSONSchema than another? I can think of the jsonschema presented during introspection, the action request schema, and the action provider input schema.

Reply:
I would try not to differentiate those unless we have to. A single schema should do it in all the cases. Either way, outside the scope of this PR.